nautobot_golden_config/utilities/helper.py

```diff
@@ -152,7 +152,7 @@ def get_repository_working_dir(
     match_rule = getattr(global_settings, f"{repo_type}_match_rule")
 
     if not match_rule:
-        return global_settings.backup_repository.first().filesystem_path
+        return getattr(global_settings, f"{repo_type}_repository").first().filesystem_path
```

Comment:
.first doesn't seem right anymore with multi-repo support.  Am I missing how it would pick the right repo if there is multiple?

Reply:
.first is when there is no match_rule, thus it cannot be multiple.

Reply:
Could it be a potential issue if multiple repos were configured with the `backup_repository` capability but only a single repo were configured as a backup in the golden config settings?

Reply:
I just tested the above and it doesn't seem to affect anything (can switch GC settings to another backup or intended repo when there are multiple available with that capability; the committed files still go to the correct repo).